### PR TITLE
DEUS VULT!!

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: Templar GuyonBroadway
add: Any knight or crusader helmet (like the chaplains) can now DEUS VULT DEUS VULT DEUS VULT DEUS VULT DEUS VULT. RE-TAKE THE HOLY LANDS!
/:cl:

[why]:

Because 1: its been too long since the chaplain took back the holy lands and 2: this will piss off doobie. 

https://youtu.be/-2uEUcThskY

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
Terrible PR, terrible joke.
But it will piss off Doobie.
+1

File: hippiestation/code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/obj/item/clothing/head/helmet/knight
+	alternate_screams = list('hippiestation/sound/voice/deus_vult.ogg')
+
+/obj/item/clothing/head/helmet/knight/equipped(mob/living/carbon/user, slot)
+	if(!ishuman(user))
+		return
+	if(slot == slot_head)
+		var/mob/living/carbon/human/H = user
+		H.reindex_screams()
```
Comment (JohnGinnane) [inline hippiestation/code/modules/clothing/head/helmet.dm:9]: It feels great to see my code being used 😊

**Review by JohnGinnane — APPROVED** [review]
Cool and good

**Comment by KayeArray** [comment]
good +1

**Comment by ghost** [comment]
god wills the pr
+1

**Comment by ghost** [comment]
and with my vote (_i guess?_) that now makes 3

**Pull request merged**